app/assets/javascripts/course/user_achievements.js

```diff
@@ -0,0 +1,12 @@
+(function($) {
+  'use strict';
+  $('#toggle-on-button' ).click(function(e) {
```

Comment:
Wait, this doesn't work with Turbolinks. You need to use delegated events on document. https://learn.jquery.com/events/event-delegation/